src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -142,7 +142,7 @@ public interface IMethodSymbol : ISymbol
         ITypeSymbol ReceiverType { get; }
 
         // PROTOTYPE(nullable-api): Document
-        Nullability ReceiverNullability { get; }
+        NullableAnnotation ReceiverNullableAnnotation { get; }
```

Comment:
>ReceiverNullableAnnotation [](start = 27, length = 26)

What's a scenario where receiver could be annotated?

Reply:
Reduced extension methods.

---
In reply to: [264934983](https://github.com/dotnet/roslyn/pull/34017#discussion_r264934983) [](ancestors = 264934983)